### ISSUE TITLE
Can't create new PhoneGap app with jquery.mobile-1.1.0-rc.1.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#73]
```
What steps will reproduce the problem?
1. Create new PhoneGap app.

What is the expected output? What do you see instead?
Create jquery.mobile-1.1.0-rc.1 project.
[Error:]
All .js and .css files in the jQuery Mobile directory 
C:\jquery.mobile-1.1.0-rc.1 must have the same version number of the form 
jquery.mobile-{version}.*

What version of the product are you using? On what operating system?
MDS Applaud 1.2
Eclipse 3.7.2
Window 7, 64bits.

Please provide any additional information below.

```

Original issue reported on code.google.com by `sugarchi...@gmail.com` on 18 Mar 2012 at 4:52

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting.

There are multiple issues triggered by new jquery mobile directory structure.

1. The *structure* files confuse the wizard and cause the error. 
Workaround: These should be moved to use the wizard and reinserted into the 
created project if the theme roller is being used.

2. jquery-mobile-1.1.0-rc.1.js and jquery.mobile-1.1.0-rc.1.min.js causes the 
JSDT validator to fail. 
Workaround: Right click on project -> Properties -> JavaScript -> Include Path 
->  -> Source -> Exclude -> Edit . See Issue 53

3. demos directory gets included in created project
Workaround: Remove it from the jquery mobile install or after project creation

4. Clean up extra jquery.mobile references in the created index.html file.

Even after making these changes,  the jquery attr() function fails in the 
sample app. We need to investigate if there is an API change or temporary 
regression in the RC.
```

Original comment by `paul.beu...@gmail.com` on 19 Mar 2012 at 4:18

**Comment by GoogleCodeExporter** [comment]
```
Thanks for reporting this! In addition to the apparent .attr() failing in the
jQuery Mobile Demo app, on Android 3.1 I am seeing a problem like jQM Issue 3723
(https://github.com/jquery/jquery-mobile/issues/3723).

Read near the end of the comments on Issue 3723 to see the path to 'latest'
jquery.mobile.js. Looks like RC2 will need to undo a lot of RC1 regression.

I'm suggesting AppLaud wait for at least jQM 1.1.0RC2, or 1.1.0 final.
```

Original comment by `libby.ba...@gmail.com` on 19 Mar 2012 at 4:38

**Comment by GoogleCodeExporter** [comment]
```
Subissues 1, 3, and 4 from comment #2 are fixed with AppLaud 1.2.8.  Leaving 
bug open for #2 and for a working jquery.mobile 1.1
```

Original comment by `paul.beu...@gmail.com` on 19 Mar 2012 at 11:37
- Added labels: **Milestone-Release1.2.9**

**Comment by GoogleCodeExporter** [comment]
```
#2 is worked around with a jsdt hack. see 
http://code.google.com/a/eclipselabs.org/p/mobile-web-development-with-phonegap/
source/detail?r=159

The jquery attr problem is avoided by rewriting the API example to use id 
instead.  
http://code.google.com/a/eclipselabs.org/p/mobile-web-development-with-phonegap/
source/detail?r=163
```

Original comment by `paul.beu...@gmail.com` on 19 Apr 2012 at 12:55
- Changed state: **Fixed**
